src/lib/github/create-project.js

```diff
@@ -1,12 +1,52 @@
 module.exports = octokit => {
-	return async ({ org, name }) => {
-		/**
-		 * Create a GitHub organisation project.
-		 *
-		 * @see https://octokit.github.io/rest.js/#api-Projects-createForOrg
-		 */
-		const result = await octokit.projects.createForOrg({ org, name });
-
-		return result.data;
+	return async ({ org, repo, user, target, name, description }) => {
+		if (org) {
```

Comment:
Worth adding a comment in here highlighting that it's guaranteed one of `org`, `repo`, or `user` will be defined.